source/administration/batch-framework.rst

```diff
@@ -0,0 +1,153 @@
+.. _minio-batch-framework:
+
+===============
+Batch Framework
+===============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. versionchanged:: MinIO RELEASE.2022-10-08T20-11-00Z or later
+
+   The Batch Framework was introduced with the ``replicate`` job type in the :mc:`mc` RELEASES.2022-10-08T20-11-00Z.
+
+Overview
+--------
+
+The MinIO Batch Framework allows you to create, manage, monitor, and execute jobs using a YAML-formatted job definition file.
+The batch jobs run directly on the MinIO deployment to take advantage of the deployment's server-side processing power without constraints of the local machine where you run the :ref:`MinIO Client <minio-client>`.
+
+A batch file can define any number of job tasks.
+MinIO does not limit on the number of job tasks that you can define in a batch file.
+
+Once started, MinIO begins processing the job immediately, depending on resources available to the deployment.
+The job runs to completion.
+If any portion of the job fails, MinIO retries the job up to the number of times defined in the job definition.
+
+A batch file only runs to completion one time.
+You can :mc-cmd:`~mc batch start` the batch file as many times as desired to complete it again.
+
+The MinIO Batch Framework supports the following job types:
+
+.. list-table:: 
+   :header-rows: 1
+   :widths: 30 70
+   :width: 100%   
+
+   * - Job Type
+     - Description
+
+   * - ``replicate``
+     - Perform a one-time replication procedure from one MinIO location to another MinIO location.
+
+Prerequisites
+~~~~~~~~~~~~~
```

Comment:
Should be H1

Reply:
Actually re-reading this section this is less of a `Prerequisites` and more of a `MinIO Batch CLI` or something.

Reply:
Renamed it to MinIO Batch CLI.